packages/airnode-protocol/package.json

```diff
@@ -28,6 +28,7 @@
     "test:gas": "REPORT_GAS=TRUE hardhat test"
   },
   "devDependencies": {
+    "@api3/chains": "https://github.com/api3dao/chains.git#new-chains",
```

Comment:
We're in the process of making some big changes to @api3/chains https://github.com/api3dao/chains/pull/1/
I will update this to use the new version as soon as possible